index.js

```diff
@@ -248,46 +249,89 @@ module.exports = {
       )
     }
 
-    // Listeners for joining groups
-    function start() {
-      ssb.metafeeds.findOrCreate((err, myRoot) => {
-        if (err) throw new Error('Could not find or create my root feed')
-        findOrCreateAdditionsFeed((err) => {
-          if (err) console.warn('Error finding or creating additions feed', err)
-        })
-
-        pull(
-          ssb.db.query(
-            where(and(isDecrypted('box2'), type('group/add-member'))),
-            live({ old: true }),
-            toPullStream()
-          ),
-          pull.filter((msg) =>
-            lodashGet(msg, 'value.content.recps', []).includes(myRoot.id)
-          ),
-          pull.asyncMap((msg, cb) => {
-            const groupId = lodashGet(msg, 'value.content.recps[0]')
-
-            ssb.box2.getGroupKeyInfo(groupId, (err, info) => {
-              if (err) {
-                console.error('Error when finding group invite for me:', err)
-                return
-              }
+    function listInvites() {
+      return pull(
+        pull.values([0]),
+        pull.asyncMap((n, cb) => {
+          ssb.metafeeds.findOrCreate((err, myRoot) => {
+            if (err) throw new Error('Could not find or create my root feed')
+
+            ssb.box2.listGroupIds((err, groupIds) => {
+              if (err) throw err
+
+              return cb(
+                null,
+                pull(
```

Comment:
Can you extract this `pull()` chain as a `const source = pull(` and then have a simple one-line `return cb(null, source)`?

Reply:
ah lmao that's simpler

Reply:
or ah first i thought you had a solution for avoiding the weird `values([0])` callback stuff. i'll do this now though

Reply:
done